deployment/base/rbac-topologyupdater/topologyupdater-clusterrole.yaml

```diff
@@ -10,6 +10,12 @@ rules:
   verbs:
   - get
   - list
+- apiGroups:
+  - ""
+  resources:
+  - nodes/proxy
+  verbs:
+  - get
```

Comment:
Kubelet checks for this to allow access?

We need to update the Helm chart (`deployment/helm/node-feature-discovery/templates/clusterrole.yaml`) too